<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatViewSettingsPage</class>
 <widget class="QWidget" name="ChatViewSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Timestamp format:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="timestampFormat">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Usage examples:&lt;/p&gt;
&lt;table cellpadding=&quot;2&quot;&gt;
&lt;tr&gt;&lt;td&gt;dd.MM.yyyy&lt;/td&gt;&lt;td&gt;21.05.2001&lt;/td&gt;&lt;/tr&gt;
&lt;tr&gt;&lt;td&gt;ddd MMMM d yy&lt;/td&gt;&lt;td&gt;Tue May 21 01&lt;/td&gt;&lt;/tr&gt;
&lt;tr&gt;&lt;td&gt;hh:mm:ss.zzz&lt;/td&gt;&lt;td&gt;14:13:09.042&lt;/td&gt;&lt;/tr&gt;
&lt;tr&gt;&lt;td&gt;h:m:s ap&lt;/td&gt;&lt;td&gt;2:13:9 pm&lt;/td&gt;&lt;/tr&gt;
&lt;/table&gt;
&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="defaultValue" stdset="0">
        <string>[hh:mm:ss]</string>
       </property>
       <property name="settingsKey" stdset="0">
        <string notr="true">TimestampFormat</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="customChatViewFont">
       <property name="text">
        <string>Custom chat window font:</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="settingsKey" stdset="0">
        <string notr="true">/QtUiStyle/Fonts/UseCustomChatViewFont</string>
       </property>
       <property name="defaultValue" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="FontSelector" name="chatViewFont" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="settingsKey" stdset="0">
        <string notr="true">/QtUiStyle/Fonts/ChatView</string>
       </property>
       <property name="defaultValue" stdset="0">
        <font>
         <family>Monospace</family>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="allowMircColors">
     <property name="toolTip">
      <string>Show colored text in the chat window</string>
     </property>
     <property name="text">
      <string>Allow colored text (mIRC color codes)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="defaultValue" stdset="0">
      <bool>true</bool>
     </property>
     <property name="settingsKey" stdset="0">
      <string notr="true">/UiStyle/AllowMircColors</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showWebPreview">
     <property name="toolTip">
      <string>Show a website preview window when hovering the mouse over a web address</string>
     </property>
     <property name="text">
      <string>Show previews of webpages on URL hover</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="defaultValue" stdset="0">
      <bool>true</bool>
     </property>
     <property name="settingsKey" stdset="0">
      <string notr="true">ShowWebPreview</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="autoMarkerLine">
     <property name="toolTip">
      <string>Set the marker line to the bottom of the current chat window when selecting another channel</string>
     </property>
     <property name="text">
      <string>Set marker line automatically when switching chats</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="settingsKey" stdset="0">
      <string notr="true">AutoMarkerLine</string>
     </property>
     <property name="defaultValue" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="autoMarkerLineOnLostFocus">
     <property name="toolTip">
      <string>Set the marker line to the bottom of the current chat window when Quassel loses focus.</string>
     </property>
     <property name="text">
      <string>Set marker line automatically when Quassel loses focus</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="settingsKey" stdset="0">
      <string>AutoMarkerLineOnLostFocus</string>
     </property>
     <property name="defaultValue" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="useCustomColors">
     <property name="title">
      <string>Custom Colors</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="settingsKey" stdset="0">
      <string notr="true">/QtUiStyle/Colors/UseChatViewColors</string>
     </property>
     <property name="defaultValue" stdset="0">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Action:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ColorButton" name="actionMsgColor">
        <property name="text">
         <string>...</string>
        </property>
        <property name="settingsKey" stdset="0">
         <string notr="true">/QtUiStyle/Colors/ActionMsg</string>
        </property>
        <property name="defaultValue" stdset="0">
         <color>
          <red>0</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Timestamp:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="ColorButton" name="timestampColor">
        <property name="text">
         <string>...</string>
        </property>
        <property name="settingsKey" stdset="0">
         <string notr="true">/QtUiStyle/Colors/Timestamp</string>
        </property>
        <property name="defaultValue" stdset="0">
         <color>
          <red>112</red>
          <green>124</green>
          <blue>112</blue>
         </color>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Channel message:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ColorButton" name="channelMsgColor">
        <property name="text">
         <string>...</string>
        </property>
        <property name="settingsKey" stdset="0">
         <string notr="true">/QtUiStyle/Colors/ChannelMsg</string>
        </property>
        <property name="defaultValue" stdset="0">
         <color>
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>134</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Highlight foreground:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="ColorButton" name="highlightColor">
        <property name="text">
         <string>...</string>
        </property>
        <property name="settingsKey" stdset="0">
         <string notr="true">/QtUiStyle/Colors/Highlight</string>
        </property>
        <property name="defaultValue" stdset="0">
         <color>
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Command message:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="ColorButton" name="commandMsgColor">
        <property name="text">
         <string>...</string>
        </property>
        <property name="settingsKey" stdset="0">
         <string notr="true">/QtUiStyle/Colors/CommandMsg</string>
        </property>
        <property name="defaultValue" stdset="0">
         <color>
          <red>150</red>
          <green>0</green>
          <blue>150</blue>
         </color>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Highlight background:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Server message:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="ColorButton" name="serverMsgColor">
        <property name="text">
         <string>...</string>
        </property>
        <property name="settingsKey" stdset="0">
         <string notr="true">/QtUiStyle/Colors/ServerMsg</string>
        </property>
        <property name="defaultValue" stdset="0">
         <color>
          <red>145</red>
          <green>100</green>
          <blue>9</blue>
         </color>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Marker line:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="ColorButton" name="markerLineColor">
        <property name="text">
         <string>...</string>
        </property>
        <property name="settingsKey" stdset="0">
         <string notr="true">/QtUiStyle/Colors/MarkerLine</string>
        </property>
        <property name="defaultValue" stdset="0">
         <color>
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Error message:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="ColorButton" name="errorMsgColor">
        <property name="text">
         <string>...</string>
        </property>
        <property name="settingsKey" stdset="0">
         <string notr="true">/QtUiStyle/Colors/ErrorMsg</string>
        </property>
        <property name="defaultValue" stdset="0">
         <color>
          <red>145</red>
          <green>100</green>
          <blue>9</blue>
         </color>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Background:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="ColorButton" name="backgroundColor">
        <property name="text">
         <string>...</string>
        </property>
        <property name="settingsKey" stdset="0">
         <string notr="true">/QtUiStyle/Colors/ChatViewBackground</string>
        </property>
        <property name="defaultValue" stdset="0">
         <color>
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="ColorButton" name="highlightBackgroundColor">
        <property name="text">
         <string>...</string>
        </property>
        <property name="settingsKey" stdset="0">
         <string notr="true">/QtUiStyle/Colors/HighlightBackground</string>
        </property>
        <property name="defaultValue" stdset="0">
         <color>
          <red>255</red>
          <green>128</green>
          <blue>0</blue>
         </color>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="useSenderColors">
     <property name="title">
      <string>Use Sender Coloring</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="settingsKey" stdset="0">
      <string notr="true">/QtUiStyle/Colors/UseSenderColors</string>
     </property>
     <property name="defaultValue" stdset="0">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="ColorButton" name="senderColor00">
          <property name="text">
           <string>...</string>
          </property>
          <property name="settingsKey" stdset="0">
           <string notr="true">/QtUiStyle/Colors/Sender00</string>
          </property>
          <property name="defaultValue" stdset="0">
           <color>
            <red>233</red>
            <green>13</green>
            <blue>127</blue>
           </color>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="ColorButton" name="senderColor02">
          <property name="text">
           <string>...</string>
          </property>
          <property name="settingsKey" stdset="0">
           <string notr="true">/QtUiStyle/Colors/Sender02</string>
          </property>
          <property name="defaultValue" stdset="0">
           <color>
            <red>179</red>
            <green>14</green>
            <blue>14</blue>
           </color>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="ColorButton" name="senderColor04">
          <property name="text">
           <string>...</string>
          </property>
          <property name="settingsKey" stdset="0">
           <string notr="true">/QtUiStyle/Colors/Sender04</string>
          </property>
          <property name="defaultValue" stdset="0">
           <color>
            <red>88</red>
            <green>175</green>
            <blue>179</blue>
           </color>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="ColorButton" name="senderColor05">
          <property name="text">
           <string>...</string>
          </property>
          <property name="settingsKey" stdset="0">
           <string notr="true">/QtUiStyle/Colors/Sender05</string>
          </property>
          <property name="defaultValue" stdset="0">
           <color>
            <red>157</red>
            <green>84</green>
            <blue>179</blue>
           </color>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="ColorButton" name="senderColor06">
          <property name="text">
           <string>...</string>
          </property>
          <property name="settingsKey" stdset="0">
           <string notr="true">/QtUiStyle/Colors/Sender06</string>
          </property>
          <property name="defaultValue" stdset="0">
           <color>
            <red>179</red>
            <green>151</green>
            <blue>117</blue>
           </color>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="ColorButton" name="senderColor07">
          <property name="text">
           <string>...</string>
          </property>
          <property name="settingsKey" stdset="0">
           <string notr="true">/QtUiStyle/Colors/Sender07</string>
          </property>
          <property name="defaultValue" stdset="0">
           <color>
            <red>49</red>
            <green>118</green>
            <blue>179</blue>
           </color>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="ColorButton" name="senderColor08">
          <property name="text">
           <string>...</string>
          </property>
          <property name="settingsKey" stdset="0">
           <string notr="true">/QtUiStyle/Colors/Sender08</string>
          </property>
          <property name="defaultValue" stdset="0">
           <color>
            <red>233</red>
            <green>13</green>
            <blue>127</blue>
           </color>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="ColorButton" name="senderColor09">
          <property name="text">
           <string>...</string>
          </property>
          <property name="settingsKey" stdset="0">
           <string notr="true">/QtUiStyle/Colors/Sender09</string>
          </property>
          <property name="defaultValue" stdset="0">
           <color>
            <red>142</red>
            <green>85</green>
            <blue>233</blue>
           </color>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="ColorButton" name="senderColor10">
          <property name="text">
           <string>...</string>
          </property>
          <property name="settingsKey" stdset="0">
           <string notr="true">/QtUiStyle/Colors/Sender10</string>
          </property>
          <property name="defaultValue" stdset="0">
           <color>
            <red>179</red>
            <green>14</green>
            <blue>14</blue>
           </color>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="ColorButton" name="senderColor11">
          <property name="text">
           <string>...</string>
          </property>
          <property name="settingsKey" stdset="0">
           <string notr="true">/QtUiStyle/Colors/Sender11</string>
          </property>
          <property name="defaultValue" stdset="0">
           <color>
            <red>23</red>
            <green>179</green>
            <blue>57</blue>
           </color>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="ColorButton" name="senderColor12">
          <property name="text">
           <string>...</string>
          </property>
          <property name="settingsKey" stdset="0">
           <string notr="true">/QtUiStyle/Colors/Sender12</string>
          </property>
          <property name="defaultValue" stdset="0">
           <color>
            <red>88</red>
            <green>175</green>
            <blue>179</blue>
           </color>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="ColorButton" name="senderColor13">
          <property name="text">
           <string>...</string>
          </property>
          <property name="settingsKey" stdset="0">
           <string notr="true">/QtUiStyle/Colors/Sender13</string>
          </property>
          <property name="defaultValue" stdset="0">
           <color>
            <red>157</red>
            <green>84</green>
            <blue>179</blue>
           </color>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="ColorButton" name="senderColor14">
          <property name="text">
           <string>...</string>
          </property>
          <property name="settingsKey" stdset="0">
           <string notr="true">/QtUiStyle/Colors/Sender14</string>
          </property>
          <property name="defaultValue" stdset="0">
           <color>
            <red>179</red>
            <green>151</green>
            <blue>117</blue>
           </color>
          </property>
         </widget>
        </item>
        <item row="1" column="7">
         <widget class="ColorButton" name="senderColor15">
          <property name="text">
           <string>...</string>
          </property>
          <property name="settingsKey" stdset="0">
           <string notr="true">/QtUiStyle/Colors/Sender15</string>
          </property>
          <property name="defaultValue" stdset="0">
           <color>
            <red>49</red>
            <green>118</green>
            <blue>179</blue>
           </color>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="ColorButton" name="senderColor01">
          <property name="text">
           <string>...</string>
          </property>
          <property name="settingsKey" stdset="0">
           <string notr="true">/QtUiStyle/Colors/Sender01</string>
          </property>
          <property name="defaultValue" stdset="0">
           <color>
            <red>142</red>
            <green>85</green>
            <blue>233</blue>
           </color>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="ColorButton" name="senderColor03">
          <property name="text">
           <string>...</string>
          </property>
          <property name="settingsKey" stdset="0">
           <string notr="true">/QtUiStyle/Colors/Sender03</string>
          </property>
          <property name="defaultValue" stdset="0">
           <color>
            <red>23</red>
            <green>179</green>
            <blue>57</blue>
           </color>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Own messages:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ColorButton" name="senderColorSelf">
          <property name="text">
           <string>...</string>
          </property>
          <property name="settingsKey" stdset="0">
           <string notr="true">/QtUiStyle/Colors/SenderSelf</string>
          </property>
          <property name="defaultValue" stdset="0">
           <color>
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>19</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QToolButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>FontSelector</class>
   <extends>QWidget</extends>
   <header>fontselector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>timestampFormat</tabstop>
  <tabstop>customChatViewFont</tabstop>
  <tabstop>allowMircColors</tabstop>
  <tabstop>showWebPreview</tabstop>
  <tabstop>autoMarkerLine</tabstop>
  <tabstop>useCustomColors</tabstop>
  <tabstop>actionMsgColor</tabstop>
  <tabstop>channelMsgColor</tabstop>
  <tabstop>commandMsgColor</tabstop>
  <tabstop>serverMsgColor</tabstop>
  <tabstop>errorMsgColor</tabstop>
  <tabstop>timestampColor</tabstop>
  <tabstop>highlightColor</tabstop>
  <tabstop>highlightBackgroundColor</tabstop>
  <tabstop>markerLineColor</tabstop>
  <tabstop>backgroundColor</tabstop>
  <tabstop>useSenderColors</tabstop>
  <tabstop>senderColor00</tabstop>
  <tabstop>senderColor01</tabstop>
  <tabstop>senderColor02</tabstop>
  <tabstop>senderColor03</tabstop>
  <tabstop>senderColor04</tabstop>
  <tabstop>senderColor05</tabstop>
  <tabstop>senderColor06</tabstop>
  <tabstop>senderColor07</tabstop>
  <tabstop>senderColor08</tabstop>
  <tabstop>senderColor09</tabstop>
  <tabstop>senderColor10</tabstop>
  <tabstop>senderColor11</tabstop>
  <tabstop>senderColor12</tabstop>
  <tabstop>senderColor13</tabstop>
  <tabstop>senderColor14</tabstop>
  <tabstop>senderColor15</tabstop>
  <tabstop>senderColorSelf</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>customChatViewFont</sender>
   <signal>toggled(bool)</signal>
   <receiver>chatViewFont</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
